sonarr/series.go

```diff
@@ -6,11 +6,89 @@ import (
+	TvRageID          int64             `json:"tvRageId,omitempty"`
+	AirTime           string            `json:"airTime,omitempty"`
+	Certification     string            `json:"certification,omitempty"`
+	CleanTitle        string            `json:"cleanTitle,omitempty"`
+	ImdbID            string            `json:"imdbId,omitempty"`
+	Network           string            `json:"network,omitempty"`
+	Overview          string            `json:"overview,omitempty"`
+	Path              string            `json:"path,omitempty"`
+	SeriesType        string            `json:"seriesType,omitempty"`
+	SortTitle         string            `json:"sortTitle,omitempty"`
+	Status            string            `json:"status,omitempty"`
+	Title             string            `json:"title,omitempty"`
+	TitleSlug         string            `json:"titleSlug,omitempty"`
+	RootFolderPath    string            `json:"rootFolderPath,omitempty"`
+	Added             *time.Time        `json:"added,omitempty"`
+	FirstAired        *time.Time        `json:"firstAired,omitempty"`
+	NextAiring        *time.Time        `json:"nextAiring,omitempty"`
+	PreviousAiring    *time.Time        `json:"previousAiring,omitempty"`
+	AddOptions        *AddSeriesOptions `json:"addOptions,omitempty"`
+	Ratings           *starr.Ratings    `json:"ratings,omitempty"`
+	Statistics        *Statistics       `json:"statistics,omitempty"`
+	Tags              []int             `json:"tags,omitempty"`
+	Genres            []string          `json:"genres,omitempty"`
+	AlternateTitles   []*AlternateTitle `json:"alternateTitles,omitempty"`
+	Seasons           []*Season         `json:"seasons,omitempty"`
+	Images            []*starr.Image    `json:"images,omitempty"`
+}
+
+// AddSeriesOptions is part of Series.
+type AddSeriesOptions struct {
+	SearchForMissingEpisodes     bool `json:"searchForMissingEpisodes"`
+	SearchForCutoffUnmetEpisodes bool `json:"searchForCutoffUnmetEpisodes,omitempty"`
+	IgnoreEpisodesWithFiles      bool `json:"ignoreEpisodesWithFiles,omitempty"`
+	IgnoreEpisodesWithoutFiles   bool `json:"ignoreEpisodesWithoutFiles,omitempty"`
+}
+
+// AlternateTitle is part of a Series.
+type AlternateTitle struct {
+	SeasonNumber int    `json:"seasonNumber"`
+	Title        string `json:"title"`
+}
+
+// Season is part of Series and Queue and used in a few places.
+type Season struct {
+	Monitored    bool        `json:"monitored"`
+	SeasonNumber int         `json:"seasonNumber"`
+	Statistics   *Statistics `json:"statistics,omitempty"`
+}
+
+// Statistics is part of Series and Queue.
+type Statistics struct {
+	SeasonCount       int        `json:"seasonCount"`
+	EpisodeFileCount  int        `json:"episodeFileCount"`
+	EpisodeCount      int        `json:"episodeCount"`
+	TotalEpisodeCount int        `json:"totalEpisodeCount"`
+	SizeOnDisk        int64      `json:"sizeOnDisk"`
+	PercentOfEpisodes float64    `json:"percentOfEpisodes"`
+	PreviousAiring    *time.Time `json:"previousAiring"`
```

Comment:
This is an output struct, making time a pointer has no value here.

Reply:
right, I can change that back, in request you'll probably omit the whole field